ProcessManager.php

```diff
@@ -184,6 +184,11 @@ class ProcessManager
     protected $timeout = 30;
 
     /**
+     * Controller port
+     */
+    public const CONTROLLER_PORT = 5500;
```

Comment:
`public` for `const` is php 7.1 only :P You can remove that, as public is default